tests/phpunit/tests/image/editor.php

```diff
@@ -18,6 +18,7 @@ public function set_up() {
 		require_once ABSPATH . WPINC . '/class-wp-image-editor.php';
 
 		require_once DIR_TESTDATA . '/../includes/mock-image-editor.php';
+		add_filter( 'image_editor_output_format', '__return_empty_array' );
```

Comment:
I think here, and everywhere else where you're adding this in a `set_up` method, you will also need to add a "revert" of this in a corresponding `tear_down` method like below, otherwise the setup becomes "polluted" between tests.
```php
remove_filter( 'image_editor_output_format', '__return_empty_array' );
```

Reply:
👍🏼 I am switching this in tests to `	remove_filter( 'image_editor_output_format', 'wp_default_image_output_mapping' );` - can re-add in tear_down

Reply:
switching to `add_filter( 'image_editor_output_format', '__return_empty_array' );` - for some reason I thought all filters were reset for test runs, but I may be wrong about that, I'll add the cleanup to be sure.

Reply:
switched throughout

Reply:
@adamsilverstein 
> for some reason I thought all filters were reset for test runs, but I may be wrong about that, I'll add the cleanup to be sure.

They do, for individual test methods. But whatever you do in any `set_up` method won't be undone, you need to revert it in a `tear_down`.

Changes look good, only right here it looks like you missed implementing the `tear_down` piece.

Reply:
added `tear_down` below